code/script.js

```diff
@@ -1,19 +1,120 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+// prompt to ask for the name and store the answer in a variable
+const guestName = prompt('Please enter your name')
+alert('Hi ' + guestName + '!')
 
 // Step 2 - Food choice
-// Your code goes here
+//  allow the user to select the type of food they want to order
+
+const foodChoice = prompt('What type of food would you like to order?\Please enter a number:\ 1 - Pizza 2 - Pasta 3 - Salad');
+
+if (foodChoice === '1') {
+  alert('You have chosen Pizza!')
+} else if (foodChoice === '2') {
+  alert('You have chosen Pasta!')
+}
+else if (foodChoice === '3') {
+  alert('You have chosen Salad!')
+}
+else {
+  alert('Invalid choice. Please select a number between 1 and 3.')
+}
 
 // Step 3 - Subtype choice
 // Your code goes here
 
+// Pizza subtype
+if (foodChoice === '1') {
+  const pizzaType = prompt('Select a Pizza type! \ Enter a number:\ 1 - Margherita 2 - Pepperoni 3 - Hawaiian')
+
+  if (pizzaType === '1') {
+    alert('You have chosen Margherita!')
+    selectedFoodType = 'Margherita'
+  } else if (pizzaType === '2') {
+    alert('You have chosen Pepperoni!')
+    selectedFoodType = Pepperoni
```

Comment:
Pepperoni is not defined - should be quotation marks around it

Reply:
Hi, @JennieDalgren made the requested changes to the JS file.